### PR TITLE
Update MetaDeploy text and translations

### PR DESCRIPTION
**Pull request by davidmreed**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Code does not reference translatable strings in its logic
- [ ] Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))
- [ ] Base axe-core JEST test included for any new LWC (No critical violations)
- [ ] CRUD/FLS is enforced in Apex
- [ ] Permission sets are updated to account for CRUD/FLS for new object metadata
- [ ] All modified classes are unit tested (Apex) at 100% coverage
- [ ] UX approval or UX not necessary
- [ ] PR contains draft release notes
- [ ] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

File: metadeploy/labels_es-MX.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
...
...rvicio.\r\n\r\n*IMPORTANTE*: Salesforce no recomienda instalar actualizaciones directamente en organizaciones de producción. En su lugar, instale el Módulo de gestión de programas en uno de sus entornos sandbox u organizaciones de desarrollador gratuitos primero.\r\n\r\nSalesforce.org Program Management Module (“PMM”) is an extensible application with open source and closed source components to address the unique core needs of program management. PMM utilizes a Salesforce.com, Inc. (“Salesforce”) managed package to provide nonprofits with a path to centralize programmatic information in order to more effectively manage and deliver programs and services. Program Management Module provides out-of-the-box translations for Spanish, French, German, Dutch, and Japanese.\r\n\r\nUna vez instalado el Módulo de gestión de programas, consulte los recursos en nuestra documentación.\r\n\r\n* [Documentación del Módulo de gestión de programas](https://powerofus.force.com/s/article/PMM-Documentation)\r\n\r\n¿Preguntas? Conviértase en miembro del grupo de clientes [Módulo de gestión de programas](https://powerofus.force.com/s/group/0F91E000000bs8XSAQ/program-management-module) en Power of Us Hub.",
```
Comment (jjbennett) [inline metadeploy/labels_es-MX.json:12]: Are we intentionally adding English into the translated paragraph?

Comment (davidmreed) [reply metadeploy/labels_es-MX.json:12]: Yes. The translated version is not yet available from SFDO Legal. (@meggraySFDO)

**Comment by davidmreed** [comment]
Please hold merging this for a few moments: I am going to add a fix for the missing translatable string in the PMM installer.

**Comment by davidmreed** [comment]
This PR now includes the missing translatable strings identified in LQA.

@AndersonTarren  @jjbennett What process needs to be followed to get those additional strings translated?

**Comment by AndersonTarren** [comment]
@davidmreed  - I'll reach out to see what we can do.

**Comment by davidmreed** [comment]
I'm closing this PR in favor of #208, which includes these updates and more.

**Pull request closed without merging**